content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,236 @@
+capacity: 20
+priority: 1
+range: [151-170]
+reset: false
+ldap: false
+location: 'fully qualified domain name of default JupyterHub server'
+replayId: 31
+varpass: false
+compile: false
+workshopImg: 'https://us-central1-grommet-designer.cloudfunctions.net/images/frederic-passeron-hpe-com/WOD-GO-101-A-simp-introduction-to-Go-programming-language.jpeg'
+badgeImg: 'https://us-central1-grommet-designer.cloudfunctions.net/images/frederic-passeron-hpe-com/go101-a-simple-introduction-to-go-programming-language.jpg'
+beta: false
+category: ['Open Source']
+duration: 4
+alternateLocation: ['fully qualified domain name of an alternate JupyterHub server']
+presenter: 'Matthew Doddler'
+role: 'FullStack developer'
+avatar: '/img/SpeakerImages/MattD.jpg'
+replayLink: 'https://hpe-developer-portal.s3.amazonaws.com/Workshops-on-Demand-Coming-Soon-Replay.mp4'
+```
+
+The following file will be used to update the **workshops table** in the database. Let's have a look at what a new entry could look like:
+
+![](/img/wod-db-go-1.png "GO 101 Workshop DB screenshot")
+
+![](/img/wod-db-go-2.png "GO 101 Workshop DB screenshot")
+
+As a contributor, Matt should be able to provide all the following details.
+
+* **ID:** A workshop ID to be used by backend server automation and Replays table to reference the associated replay video of the workshop (automatically created at the import of the wod.yml file process)
+* **name:** The workshop's name as it will will be displayed on the registration portal
+* **notebook:** Name of the folder containing all the workshop's notebooks (automatically created at the import of the wod.yml file process)
+* **description:** The workshop's abstract as it will will be displayed on the registration portal
+* **avatar, role and replayLink** are superseded by entries in the replay table. I will explain later
+* **R﻿eplayId:** This entry links the dedicated replay video to the workshop. it enables the presence of the replay in the learn more page of the workshop (automatically created at the import of the wod.yml file process)
+* **Category:** The workshops' registration portal proposes several filters to display the catlog's content. You can view all workshops, the most poular ones, or by category. Use this field to sort workshops accordingly
+* **D﻿uration:** All workshops are time bombed. You will define here the time allocated to perform the workshop
+* **A﻿ctive:** Tag to set to enable visibility of the workshop's tile in the registration portal
+* **W﻿orkshopImg:** As part of the lifecycle of the workshop, several emails are sent to the student. A workshop image is embedded in the first emails
+* **session type:** Workshops-on-Demand by default (automatically created at the import of the wod.yml file process)
+
+The following fields are required by the infrastructure. And I will work as the infrastructure Admin with Matt to define them.
+
+* **capacity:** The number of maximum concurrent students allowed to take on the workshop
```

Comment:
A quick question, Fred. Why are some of these bullets using an initial capital letter and others aren't? Since they are at the beginning of a bullet, I would capitalize each one's initial letter; i.e. Capacity, Range, Location, Compile, Beta, etc.

But if you have a specific reason for it, like the developers need to see it as a code name or something, then I guess it's okay.